nodeEditor/src/diagram/display/inputDisplayManager.ts

```diff
@@ -46,15 +45,19 @@ export class InputDisplayManager implements IDisplayManager {
 
     public getHeaderText(block: NodeMaterialBlock): string {
         let inputBlock = block as InputBlock;
-        let name = `${inputBlock.name} (${StringTools.GetBaseType(inputBlock.output.type)})`;
+        let name = `${inputBlock.name} (${this.GetBaseType(inputBlock.output.type)})`;
 
         if (inputBlock.isAttribute) {
-            name = StringTools.GetBaseType(inputBlock.output.type);
+            name = this.GetBaseType(inputBlock.output.type);
         }
 
         return name;
     }
 
+    public GetBaseType(type: NodeMaterialBlockConnectionPointTypes): string {

```

Comment:
You should either make this method static or have the first letter of its name be lowercase.